packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.component.tsx

```diff
@@ -303,33 +310,38 @@ const VisitTable: React.FC<VisitTableProps> = ({ showAllEncounters, visits, pati
                       <TableExpandedRow className={styles.expandedRow} colSpan={headers.length + 2}>
                         <>
                           <EncounterObservations observations={visits[index].obs} />
-                          {visits[index]?.form?.uuid && (
-                            <Button
-                              kind="ghost"
-                              onClick={() => {
-                                launchWorkspace(
-                                  visits[index].form.uuid,
-                                  visits[index].visitUuid,
-                                  visits[index].id,
-                                  visits[index].form.display,
-                                  visits[index].visitTypeUuid,
-                                  visits[index]?.visitStartDatetime,
-                                  visits[index]?.visitStopDatetime,
-                                );
-                              }}
-                              renderIcon={(props) => <Edit size={16} {...props} />}
-                            >
-                              {t('editThisEncounter', 'Edit this encounter')}
-                            </Button>
-                          )}
-                          {visits[index]?.form?.display && (
-                            <Button
-                              kind="danger--ghost"
-                              onClick={() => handleDeleteEncounter(visits[index].id, visits[index].form.display)}
-                              renderIcon={(props) => <TrashCan size={16} {...props} />}
-                            >
-                              {t('deleteThisEncounter', 'Delete this encounter')}
-                            </Button>
+                          <Button kind="ghost" renderIcon={(props) => <Document size={16} {...props} />}>
+                            {t('goToThisEncounter', 'Go to this encounter')}
+                          </Button>
```

Comment:
a user without access to the encounter will see `Go to this encounter`. Is `Document` the right icon to display here?